packages/next-bundle-analyzer/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = ({ enabled = true } = {}) => (nextConfig = {}) => {
             reportFilename: options.isServer
               ? '../analyze/server.html'
               : './analyze/client.html',
+            ...bundleAnalyzerOptions
```

Comment:
instead, a callback could be used to:

`(nextConfig: NextConfig) => BundleAnalyzerOptions`

this would actually be better to support `options.isServer`

Reply:
implemented in next iteration :wave: